scripts/generate-icons.py

```
@@ -0,0 +1,200 @@
+    try:
+        font = ImageFont.truetype(FONT_PATH, 90)
+        word = "PERCOLATOR"
+        tmp = ImageDraw.Draw(Image.new("RGBA", (1, 1)))
+        bb = tmp.textbbox((0, 0), word, font=font)
+        word_layer = Image.new("RGBA", (W, H), (0, 0, 0, 0))
+        wd = ImageDraw.Draw(word_layer)
+        wx = cx - (bb[2] - bb[0]) // 2 - bb[0]
+        wy = cy + font_size // 2 + 40
+        wd.text((wx, wy), word, font=font, fill=(*TEXT, 200))
+        img = Image.alpha_composite(img, word_layer)
+    except Exception as e:
```

Ruff (0.15.4)

[warning] Do not catch blind exception: `Exception`

(BLE001)